src/js/dateHowLongAgo.js

```diff
@@ -0,0 +1,23 @@
+// change date into how long ago that date is relative to when view page
+export function dateHowLongAgo(date) {
+  function render(n, unit) {
```

Comment:
nit: can we rename render to something more semantic? like `formatDate` or `dateFormatHelper`?

Reply:
Can do!